src/profile-logic/processed-profile-versioning.js

```diff
@@ -2372,6 +2372,16 @@ const _upgraders = {
       delete stackTable.subcategory;
     }
   },
+  [54]: (profile) => {
+    // The `implementation` column was removed from the frameTable. Modern
+    // profiles from Firefox use subcategories to represent the information
+    // about the JIT type of a JS frame.
+    for (const thread of profile.threads) {
+      delete thread.frameTable.implementation;
+    }
```

Comment:
I was wondering about converting this data into subcategory-based data, for these profiles that don't have subcategories.
But that's probably also very old profiles, so maybe not worth it.
What do you think?

Reply:
For example:

[production](https://profiler.firefox.com/public/280e10708c0660695ef5af145c53179d29d9796c/calltree/?globalTrackOrder=0w4&hiddenGlobalTracks=1&thread=4&timelineType=category&v=10)
[deploy preview](https://deploy-preview-5370--perf-html.netlify.app/public/280e10708c0660695ef5af145c53179d29d9796c/calltree/?globalTrackOrder=0w4&hiddenGlobalTracks=1&thread=4&timelineType=category&v=10)

Reply:
I've added a commit on top which migrates the information. It doesn't attempt to detect interpreter C++ frames or other C++ frames. This would be more work and I don't think it would be worth doing.